Packages/com.unity.inputsystem/InputSystem/Plugins/HID/HID.cs

```diff
@@ -932,7 +932,7 @@ public HIDDeviceDescriptor Finish()
                     usage = usage,
                     usagePage = usagePage,
                     elements = m_Elements != null ? m_Elements.ToArray() : null,
-                    collections = m_Collections != null ? m_Collections.ToArray() : null,
+                    collections =  null,
```

Comment:
Please add a ////TODO or something like that to make it clear this isn't how it's meant to be.